src/scene/scene.js

```diff
@@ -396,6 +396,15 @@ class Scene extends EventHandler {
     set envAtlas(value) {
         if (value !== this._envAtlas) {
             this._envAtlas = value;
+
+            // make sure required options are set up on the texture
+            if (value) {
```

Comment:
Actually mipmaps must also be disabled, but this should be done at load time.

Reply:
do we have some asset options for that do you know?